recipes/connectome-workbench/build.sh

```
@@ -1,15 +1,20 @@
-#!/bin/bash
-
-export PKG_CONFIG_PATH=${PREFIX}/lib/pkgconfig \
- OPENMP_HEADER_DIR=${PREFIX}/include \
- OPENMP_LIB_DIR=${PREFIX}/lib \
- FREETYPE_DIR=${PREFIX}
+#!/usr/bin/env bash
 
 mkdir build
 cd build
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)